setup.py

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env python3
 # encoding=utf-8
 
+import sys
+
 from setuptools import setup
 
-setup(setup_requires=['pbr', 'appdirs'], pbr=True, data_files=[('config', ['bdfr/default_config.cfg'])])
+if sys.version_info >= (3,9):
+    setup(setup_requires=['pbr', 'appdirs'], pbr=True, data_files=[('config', ['bdfr/default_config.cfg'])])
+else:
+    raise RuntimeError("This package requres Python 3.9+")
```

Comment:
requires

Reply:
Did your message get cut off? This makes no sense

Reply:
There is a typo in the error message @Serene-Arc